wemix/bind/structs.go

```diff
@@ -438,7 +460,6 @@ type InitEnvStorage struct {
 	BLOCK_GASLIMIT                           *big.Int
 	BASE_FEE_MAX_CHANGE_RATE                 *big.Int
 	GAS_TARGET_PERCENTAGE                    *big.Int
-	Options                                  map[string]*big.Int
 }
```

Comment:
1. Rmove duplicate code using reflect and tag
2. Easily check method names and variable names
3. Process all without missing members

```
type InitEnvStorage struct {
	BLOCKS_PER                               *big.Int `method:"blocksPer"`
	BALLOT_DURATION_MIN                      *big.Int `method:"ballotDurationMin"`
	BALLOT_DURATION_MAX                      *big.Int `method:"ballotDurationMax"`
	STAKING_MIN                              *big.Int `method:"stakingMin"`
	STAKING_MAX                              *big.Int `method:"stakingMax"`
	MAX_IDLE_BLOCK_INTERVAL                  *big.Int `method:"MaxIdleBlockInterval"`
	BLOCK_CREATION_TIME                      *big.Int `method:"blockCreationTime"`
	BLOCK_REWARD_AMOUNT                      *big.Int `method:"blockRewardAmount"`
	MAX_PRIORITY_FEE_PER_GAS                 *big.Int `method:"maxPriorityFeePerGas"`
	BLOCK_REWARD_DISTRIBUTION_BLOCK_PRODUCER *big.Int `method:"blockRewardDistributionBlockProducer"`
	BLOCK_REWARD_DISTRIBUTION_STAKING_REWARD *big.Int `method:"blockRewardDistributionStakingReward"`
	BLOCK_REWARD_DISTRIBUTION_ECOSYSTEM      *big.Int `method:"blockRewardDistributionEcosystem"`
	BLOCK_REWARD_DISTRIBUTION_MAINTENANCE    *big.Int `method:"blockRewardDistributionMaintenance"`
	MAX_BASE_FEE                             *big.Int `method:"maxBaseFee"`
	BLOCK_GASLIMIT                           *big.Int `method:"blockGasLimit"`
	BASE_FEE_MAX_CHANGE_RATE                 *big.Int `method:"baseFeeMaxChangeRate"`
	GAS_TARGET_PERCENTAGE                    *big.Int `method:"gasTargetPercentage"`
}
```

```
func (cfg InitEnvStorage) Args() (names [][32]byte, values []*big.Int) {
	v := reflect.ValueOf(cfg)
	t := reflect.TypeOf(cfg)
	numField := v.NumField()
	names = [][32]byte{}
	values = []*big.Int{}
	for i := 0; i < numField; i++ {
		value, ok := v.Field(i).Interface().(*big.Int)
		if ok && value != nil && value.Sign() > 0 {
			tag := t.Field(i).Tag.Get("method")
			names = append(names, crypto.Keccak256Hash([]byte(tag)))
			values = append(values, value)
		}
	}
	return
}
```